libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/utils/ValidationUtilsTest.java

```diff
@@ -0,0 +1,102 @@
+package com.mapbox.services.android.navigation.v5.utils;
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
+import com.mapbox.api.directions.v5.models.RouteOptions;
+import com.mapbox.core.constants.Constants;
+import com.mapbox.geojson.Point;
+import com.mapbox.services.android.navigation.v5.BaseTest;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.MissingFormatArgumentException;
+
+public class ValidationUtilsTest extends BaseTest {
+
+  private static final String DIRECTIONS_WITHOUT_VOICE_INSTRUCTIONS = "directions_v5_no_voice.json";
+
+  @Test(expected = MissingFormatArgumentException.class)
+  public void validDirectionsRoute_isInvalidWithNullRouteOptions() throws Exception {
+    DirectionsRoute route = buildTestDirectionsRoute(DIRECTIONS_WITHOUT_VOICE_INSTRUCTIONS);
+    route = route.toBuilder().routeOptions(null).build();
```

Comment:
What about extracting the `null` into an explaining variable to make clear what we're specifically testing?